### DISCUSSION TITLE
Manifest V3 transition timeline on Microsoft Edge

### DISCUSSION THREAD

**Discussion by sonakshimore** [microsoft/MicrosoftEdge-Extensions#49] [Q&A] [answered]
With the adoption of MV3 standards among all major browsers that allow browser extensions at the client-side, we understand that this is the same with Microsoft Edge as well, as mentioned [here](https://docs.microsoft.com/en-in/microsoft-edge/extensions-chromium/developer-guide/manifest-v3).

However, as with Firefox, browsers may choose to adopt them incompletely and with few changes. We understand that even Chromium-like browsers can deviate from the Chromium standards. I had a few questions regarding how Microsoft views this transition and what would be the next steps from here.

Thus, it would be really great if you could provide your insights on any or all of the following points: 
1. Would MS Edge adopt the exact standards proposed by the Chromium ecosystem such as with functionalities like header-based modifications in the coming days? 
2. What would the general timeline be for this in general? 
3. Does this also translate to the fact that existing standards with MV2 standards would not be allowed to operate any further, as with the timeline with Chromium?

**Comment by sonakshimore** [comment, marked as answer]
To answer your questions:

> 1. Would MS Edge adopt the exact standards proposed by the Chromium ecosystem such as with functionalities like header-based modifications in the coming days?

Yes, that is correct.

> 2. What would the general timeline be for this in general?
> 3. Does this also translate to the fact that existing standards with MV2 standards would not be allowed to operate any further, as with the timeline with Chromium?

The Microsoft documentation [Overview and timelines for migrating to Manifest V3](https://docs.microsoft.com/en-in/microsoft-edge/extensions-chromium/developer-guide/manifest-v3) outlines the timeline for: 
1. Migrating your extension to Manifest V3
2. When Manifest V2 extensions will not be supported by Microsoft Edge

Here's an overview of the migration timelines, as seen on our official documentation shared above:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/85096285/189859933-12e0b650-5a7a-40f4-9edf-8181f0d301da.png">

**Reply by allentown521** [reply]
Nearly four years have passed, and the timeline still doesn't seem to have been updated.
